sdk/template-LLC/test-resources.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+        },
+        "tenantId": {
+            "type": "string",
+            "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+            "metadata": {
+                "description": "The tenant ID to which the application and resources belong."
+            }
+        },
+        "testApplicationOid": {
+            "type": "string",
+            "defaultValue": "b3653439-8136-4cd5-aac3-2a9460871ca6",
+            "metadata": {
+                "description": "The client OID to grant access to test resources."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The location of the resource. By default, this is the same as the resource group."
+            }
+        }
+    },
+    "variables": {
```

Comment:
I like that in the Azure.Template file they have some service-related examples here.  How much of these non-LLC-specific files could we reuse from that example?

Reply:
We have added customized client [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/src/MiniSecretClient.cs) in the Azure.Template in order to add those test cases. I thought of making template LLC project simple, without any customization code. Otherwise when people will copy the project, they will copy the customized code as well, which is not needed to generate basic code for LLC.

Reply:
I'm thinking of adding all the details in LLC quick start guide on how to add tests and link [this](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#live-testing).